src/components/settings/settings-form/settings-form.js

```diff
@@ -21,6 +21,7 @@ export default class SettingsForm extends Component {
       invalid: PropTypes.bool,
       pristine: PropTypes.bool,
     }).isRequired,
+    lastMenu: PropTypes.element,
```

Comment:
Why this prop isn't required?

Reply:
@Vladyslav-Velytskyi , because there are cases when last menu should not be rendered